docs/apis-tools/community-clients/micronaut.md

```diff
@@ -7,7 +7,7 @@ The Micronaut integration is a community extension allowing you to leverage Zeeb
 
 The integration provides a wrapper around the [Zeebe Java Client](/apis-tools/java-client/index.md), and is preconfigured with sensible defaults so you can get started with minimal configuration. Add a dependency, implement a worker, and add your credentials in your Micronaut project.
 
-The Micronaut Framework is known for its efficient use of resources. Native images created with [GraalVM](https://www.graalvm.org/) reduce startup times to milliseconds.
+The Micronaut Framework is known for its efficient use of resources. Built-in images created with [GraalVM](https://www.graalvm.org/) reduce startup times to milliseconds.
```

Comment:
I was just discussing this on the DevRel Collective Slack. I don't know if this is the best substitution here, but I can't think of a better one. In the context of images this is still a well-known term. Could you do an internal poll in ask-documentation or an external community poll and get more feedback on this?

Reply:
You may find [this page](https://www.graalvm.org/22.0/reference-manual/native-image/) interesting given the context here.

As this is used in code snippets and defined by GraalVM, this may have to remain an exception.

Reply:
Noted in the Confluence page as an exception 👍